Dapper/DynamicParameters.cs

```diff
@@ -397,10 +397,10 @@ static void ThrowInvalidChain()
             }
             while (diving is not null);
 
-            var dynamicParamName = string.Concat(names.ToArray());
+            var dynamicParamName = string.Concat(names);
 
             // Before we get all emitty...
-            var lookup = string.Join("|", names.ToArray());
+            var lookup = string.Join("|", names);

```

Comment:
A "char" vs "string" can be used in `Join()` call.

Reply:
@mgravell With lots of pull requests in the queue, are we going to have a new Dapper release any time soon? And, if you or your team needs assistance, how can it be possible?